src/js/classes/AjaxBootstrapSelectList.js

```diff
@@ -348,6 +348,13 @@ AjaxBootstrapSelectList.prototype.setStatus = function (status) {
     status = status || '';
     if (status.length) {
         this.$status.html(status).show();
+
+        // Add status' height to dropdown maxHeight to prevent obfuscation when
+        // options list is longer than the maxHeight set on the dropdown
+        var maxHeight = this.$status.parent('.dropdown-menu').css('maxHeight');
+        this.$status.parent('.dropdown-menu').css({
+            maxHeight: parseInt(maxHeight, 10) + this.$status.outerHeight() + 'px'
```

Comment:
Not a fan of styling via jQuery/JavaScript. Instead, this should likely add a class on the dropdown menu that allows it to calculate the `max-height` via CSS instead, for example (just typed this here, not sure if this is even remotely correct):

```less
.dropdown-menu.has-status {
  max-height: (@dropdown-max-height - @dropdown-status-height);
}
```